# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/plocate/hotfix-stat-redefinition.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Avoid redefinition conflicts between sys/stat.h and linux/stat.h.

  - NoTag <notag@t2sde.org>

--- plocate-1.1.22/io_uring_engine.h.vanilla	2024-04-05 13:33:53.356396188 +0200
+++ plocate-1.1.22/io_uring_engine.h	2024-04-05 13:34:19.229397202 +0200
@@ -7,7 +7,6 @@
 #include <string_view>
 #include <sys/socket.h>
 #include <sys/types.h>
-#include <linux/stat.h>
 
 struct io_uring_sqe;
 #ifndef WITHOUT_URING
